src/controller/python/OpCredsBinding.cpp

```diff
@@ -492,7 +502,10 @@ PyChipError pychip_OpCreds_AllocateController(OpCredsContext * context, chip::Co
 
     // Initialize device attestation verifier
     const chip::Credentials::AttestationTrustStore * testingRootStore = GetTestFileAttestationTrustStore(paaTrustStorePath);
-    chip::Credentials::DeviceAttestationVerifier * dacVerifier        = chip::Credentials::GetDefaultDACVerifier(testingRootStore);
+    chip::Credentials::DeviceAttestationRevocationDelegate * dacRevocationDelegate =
+        GetAttestationRevocationDelegate(dacRevocationSetPath);
+    chip::Credentials::DeviceAttestationVerifier * dacVerifier =
+        chip::Credentials::GetDefaultDACVerifier(testingRootStore, dacRevocationDelegate);
```

Comment:
Now I see that coupling the DAC revocation delegate to the default DAC verifier is what causes you to have to do this. It should still be possible to UPDATE the config of the DefaultDACVerifier, without needing to initially provide an instance of DAC revocation delegate.  Suggest passing nullptr here.

Reply:
dynamic cast is failing so added an API which returns the pointer to the instance of type `DefaultDACVerifier`

Reply:
Added a virtual function in `DeviceAttestationVerifier::TrySettingRevocationDelegate()` which returns `NOT_IMPLEMENTED` and has an implementation in `DefaultDACVerifier`.